### DISCUSSION TITLE
add llava 1.6

### DISCUSSION THREAD

**Discussion by kalle07** [starik222/BooruDatasetTagManager#121] [Ideas]
is that possible ? and it would be great that you write the aprox GB behind all models

**Comment by kalle07** [comment]
no ? 
https://huggingface.co/liuhaotian/llava-v1.6-mistral-7b/tree/main
it give also 13b version

**Reply by starik222** [reply]
I don't understand why this is needed? Llava is a language model similar to ChatGPT. What function can it perform in the tag editor?

**Reply by kalle07** [reply]
I thought since it is so small and can be recognized images in llm. maybe it works here too ?

**Reply by JS3910** [reply]
Llava is a multi-modal language model, it can accept images as input. Models like that are much better than the older types (CLIP/BLIP/ViT/etc) at generating free-form prose descriptions of images. They can probably also do very good at tagging but certainly isn't something you can just "drop in".
